src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -108,6 +108,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@PaulHigin, your last commit had 40 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

ForEach-Object -Parallel Constrained Language Tests.Foreach-Object -Parallel must run in ConstrainedLanguage mode under system lock down
```powershell
Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\ConstrainedLanguageRestriction.Tests.ps1: line 890
890:                 $results = 1..1 | ForEach-Object -Parallel -ScriptBlock { $ExecutionContext.SessionState.LanguageMode }

```
ForEach-Object -Parallel Functional Tests.Verifies job queuing and throttle limit
```powershell
Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1: line 236
236:         $job = 1..4 | ForEach-Object -Parallel -ScriptBlock { Start-Sleep 60 } -AsJob -ThrottleLimit 2

```
ForEach-Object -Parallel Functional Tests.Verifies jobs work with Receive-Job -AutoRemove parameter
```powershell
Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1: line 260
260:         $job = 1..4 | ForEach-Object -Parallel -AsJob -ScriptBlock { "Hello:$_" }

```
ForEach-Object -Parallel Functional Tests.Verifies parallel task queuing
```powershell
Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1: line 267
267:         $results = 10..1 | ForEach-Object -Parallel -ScriptBlock { Start-Sleep 1; $_ } -ThrottleLimit 5

```
ForEach-Object -Parallel Functional Tests.Verifies timeout and throttle parameters
```powershell
Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1: line 279
279:         $results = 1..2 | ForEach-Object -Parallel { "Output $_"; Start-Sleep -Seconds 60 } -TimeoutSeconds 5 -ThrottleLimit 1 2>&1

```